plan/physical_plan_builder.go

```diff
@@ -272,6 +272,10 @@ func isCoveringIndex(columns []*model.ColumnInfo, indexColumns []*model.IndexCol
 }
 
 // convert2PhysicalPlan implements LogicalPlan convert2PhysicalPlan interface.
+// If no available index matches the required property, the returned physicalPlanInfo
+// will be table scan and has cost of MaxInt64, but it's OK because parent will call
+// convert2PhysicalPlan again with empty required property, so the plan with the lowest
+// cost will be chosen.
```

Comment:
If there are no index that matches the required property, the returned physicalPlanInfo will be table scan and has the cost of MaxInt64. But this can be ignored because the parent will call convert2PhysicalPlan again with the empty required property, the plan with the lowest cost will be chosen.